src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/TaskInfo.java

```diff
@@ -116,6 +119,11 @@
   @ApiModelProperty("Actions that can be requested for this task")
   private Map<String, Action> actions;
 
+  @ApiModelProperty("Instructions for parameterising this task")
```

Comment:
does this mean we can drop the problemStatement field?

Reply:
No, because we still want that as a more generic description for test setters (it could include explicit information about the parameterisation)

Reply:
Ah ok. That makes sense.